lib/auxiliars/helper.rb

```diff
@@ -65,7 +65,7 @@ def calculate_shipping_deadline(days, saturday, date = Date.today)
     else
       days.to_i.times do
         date += 1.days
-        date += 1.days if deadline.sunday? || deadline.saturday?
+        date += 1.days if date.sunday? || date.saturday?
```

Comment:
💅🏾 Testei aqui e teve o mesmo comportamento, o que acha?
```suggestion
        date += 1.days if date.on_weekend?
```

Reply:
Boa, vou alterar aqui

Reply:
Alterado @laerte-guimaraes